build-support/bin/_generate_all_lockfiles_helper.py

```diff
@@ -122,10 +121,6 @@ def jvm(cls, tool: type[JvmToolBase], *, backend: str | None = None) -> DefaultT
         ScroogeSubsystem, backend="pants.backend.experimental.codegen.thrift.scrooge.scala"
     ),
     DefaultTool.jvm(AvroSubsystem, backend="pants.backend.experimental.codegen.avro.java"),
-    DefaultTool(
-        "scalac-plugins",
-        (f"--scalac-plugins-global-lockfile={Scalac.default_plugins_lockfile_path}",),
```

Comment:
Does this need to be deleted?

Reply:
Not sure what you mean. The option was deleted, as well as all of its usages.

Reply:
But not the file: #15178